### ISSUE TITLE
Click on user's name in Nima/Flare notifications

### ISSUE THREAD

**Issue by neurowave** [2d-inc/support#66]
Should be able to click on name here so it takes to a user's profile:
![image 2018-11-07 at 7 36 55 pm](https://user-images.githubusercontent.com/1817068/48176379-9a00a100-e2c4-11e8-85e1-e221a74b7261.png)

**Comment by neurowave** [comment]
Was hoping I could implement this one myself, but it looks like we're not exposing the user's username in the data! Need help from @luigi-rosso or @umberto-sonnino.

Btw, this should be implemented in both Nima and Flare.

**Comment by neurowave** [comment]
This is a nitpick, consider it super low priority, but it would be nice if we could get the user's name separately from the "file" Part of the text, so we could make just the name the link.

So, in the attached example, only the word "JcToon" would underline on hover, and the "'s File" part would remain not underlined.

Makes it clearer that it's a link to the user's profile. Again, super low priority.

![screen recording 2018-11-12 at 01 35 pm](https://user-images.githubusercontent.com/1817068/48376432-c713d180-e67f-11e8-83ef-10a81a00b695.gif)

**Comment by luigi-rosso** [comment]
Fixed this, just needs help with styling. It needs to be embedded in an extra span to allow customizing the title of the alert.

<img width="567" alt="screen shot 2018-11-12 at 2 13 34 pm" src="https://user-images.githubusercontent.com/454182/48378370-3f30c600-e685-11e8-9940-1e13cc18a7b8.png">

**Comment by neurowave** [comment]
Fixed, although I can't figure out how to give the new link a className. It looks like it's doing something different from the wrapLink function we were using before (and for the avatar link). I managed to target it using a [CSS adjacent sibling selector ](https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_selectors), but I'd like to eventually understand what we're doing here better so I can make it more robust!

![screen recording 2018-11-12 at 03 37 pm](https://user-images.githubusercontent.com/1817068/48381542-2e864d00-e691-11e8-8c89-785d4eb6c2df.gif)
